### PR TITLE
Fix non existant ayat

### LINKED ISSUE #468: Non-existant ayat should return 404 but returns 500
Steps to reproduce:
- Construct a non-existant ayat URL, e.g. https://quran.com/23/150

Actual behavior: 
- 500 status code
- Generic "Internal server error" page

Expected behavior:
- 404 status code
- Friendly "Ayat not found" error page

### PR DESCRIPTION
**Pull request by aslamhadi**
This should fix #468 

screen shot 2017-03-25 at 1 51 53 pm

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32815

**Comment by sabeurthabti** [comment]
I've approved it. A nice to have, if you have time is to do the locale translate if possible, google translate should be enough.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32816

**Comment by aslamhadi** [comment]
@sabeurthabti added translation based on Google Translate in daba152

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32817

File: src/containers/Surah/index.js [hunk 1]
```diff
@@ -225,6 +225,17 @@ class Surah extends Component {
     return `${chapter.info ? chapter.info.shortDescription : ''} This Surah has ${chapter.versesCount} verses and resides between pages ${chapter.pages[0]} to ${chapter.pages[1]} in the Quran.`; // eslint-disable-line max-len
   }
 
+  renderNoAyah() {
+    const { isLoading } = this.props;
+
+    const noAyah = (<div className="text-center">
```
Comment (aslamhadi) [inline src/containers/Surah/index.js]: updated

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32818

File: src/containers/Surah/index.js (hunk 1 above)
Comment (aslamhadi) [inline src/containers/Surah/index.js]: updated

File: src/redux/modules/fontFaces.js [hunk 2]
```diff
@@ -9,13 +9,15 @@ export default function reducer(state = {}, action = {}) {
       const verses = action.result.entities.verses;
       const classNames = {};
 
-      Object.keys(verses).forEach((ayahId) => {
-        const verse = verses[ayahId];
+      if (verses !== undefined) {
```
Comment (sabeurthabti) [inline src/redux/modules/fontFaces.js]: agreed

Comment (aslamhadi) [inline src/redux/modules/fontFaces.js]: screen shot 2017-03-26 at 7 25 09 pm

no, `if(!verses)` does not work. I think if we want to catch `null` and `undefined` value, we should use this one:

`if (verses !== undefined && verses !== null)`

Comment (sabeurthabti) [inline src/redux/modules/fontFaces.js]:
```js
if(verses) {
...
}
```
should be enough, it will check for null, undefined and falsey

Comment (aslamhadi) [inline src/redux/modules/fontFaces.js]: done in 44e74c2

File: src/locale/tr.js [hunk 3]
```diff
@@ -76,6 +76,8 @@ export default {
     'nav.navigate': 'Bağlantılar', // there is no exact translation for Navigate in Turkish
     'nav.legacySite': 'Eski Siteye Git Quran.com',
 
-    'login.message': 'Tüm yer işaretlerinizi, notlarınızı ve etkinliklerinizi saklamak için Quran.com\'a giriş yapın.'
+    'login.message': 'Tüm yer işaretlerinizi, notlarınızı ve etkinliklerinizi saklamak için Quran.com\'a giriş yapın.',
+
+    'chapter.index.ayahNotFound': 'Ayet bulunamadı'
```
Comment (naveed-ahmad) [inline src/locale/tr.js]: `ayah.notFound` is more appropriate key for this IMO.

**Comment by naveed-ahmad** [comment]
rebuild

File: src/locale/tr.js (hunk 3 above)
Comment (aslamhadi) [inline src/locale/tr.js]: updated

**Comment by sabeurthabti** [comment]
what happened to jenkins ?

**Comment by ahmedre** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32819

**Comment by ahmedre** [comment]
jazakAllah khairan, fixed it

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32821

**Pull request merged**